### PR TITLE
[ja] add Japanese translation of quick start of collector

### PR DESCRIPTION
**Pull request by ymotongpoo**
Add Japanese translation of [Quick Start](https://opentelemetry.io/docs/collector/quick-start/) page

### CODE REVIEW

**Comment by ymotongpoo** [comment]
@svrnm @chalin can you figure out why component owner check fails with this PR?

**Comment by svrnm** [comment]
> @svrnm @chalin can you figure out why component owner check fails with this PR?

they fail for all PRs because we are lacking permissions, I am currently waiting for the TC to help us with fixing that.

File: content/ja/docs/collector/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: クイックスタート
+cSpell:ignore: docker dokey dpkg okey telemetrygen
+description: コレクターをセットアップとテレメトリーの収集をすぐに始めてみましょう！
+weight: 1
+cSpell:ignore: gobin
```
Comment (chalin) [inline content/ja/docs/collector/quick-start.md:6]: There should be only one `cSpell:ignore` front matter key.

Comment (chalin) [reply content/ja/docs/collector/quick-start.md:6]: Also, remove any keys that don't apply anymore.

Comment (ymotongpoo) [reply content/ja/docs/collector/quick-start.md]: @chalin thank you for reviewing. I removed unused fields from metadata.

**Comment by chalin** [comment]
@ymotongpoo - is this a DRAFT because of the GH check failure? If so, you can remove the draft status.

**Review by chalin — APPROVED** [review]

**Comment by svrnm** [comment]
@open-telemetry/docs-ja-approvers please take a look, we can only merge the PR with a review from a native speaker

**Comment by svrnm** [comment]
@open-telemetry/docs-ja-approvers please take a look

**Comment by ymotongpoo** [comment]
@chalin this change was failed due to the fail of spelling checker for `pt` page. Could you take a look at the spell checker action configuration?

**Comment by svrnm** [comment]
> @chalin this change was failed due to the fail of spelling checker for `pt` page. Could you take a look at the spell checker action configuration?

I have a PR in place to fix that. Nevertheless we would need someone to review the translation.

**Review by katzchang — APPROVED** [review]

**Comment by ymotongpoo** [comment]
@open-telemetry/docs-approvers Thank you @chalin and @svrnm for fixing the link check action. Now this PR is good to merge.

**Review by chalin — APPROVED** [review]
LGTM

**Pull request merged**